### DISCUSSION TITLE
Lazy update fails due to doc/tags

### DISCUSSION THREAD

**Discussion by doctorfree** [loctvl842/monokai-pro.nvim#21] [Q&A] [answered]
Hi, I am enjoying the monokai-pro colorscheme but recently experienced a problem with fresh installs thinking they needed to be updated. I use the Lazy plugin manager and nightly Neovim build. I'm opening a discussion rather than an issue as I don't really know what is going on yet.

The problem surfaces when I open Neovim and Lazy informs me that monokai-pro has an update. However, running `Lazy update` produces an error informing me that it cannot perform the update due to `doc/tags`. If I remove `~/.local/share/nvim-lazyman/lazy/monokai-pro.nvim/doc/tags` and retry the `Lazy update` it succeeds and a new `doc/tags` is created (it's the same).

Is it possible that the `doc/tags` in the repository is out of sync with the input used to create it? Or, should `doc/tags` be in the repository if it is dynamically created? It is, of course, quite possible the problem is on my end. In addition to a nightly Neovim build I am using a non-standard configuration location and `NVIM_APPNAME` environment variable. So, I am just asking for help resolving this. Thanks for any suggestions.

**Comment by loctvl842** [comment, marked as answer]
Ok I'll try to figure it out. I've just release new version of monokai-pro colorscheme. Please try it out and leave a comment.

**Reply by doctorfree** [reply]
I tried again this morning with the same results. Lazy thinks monokai-pro has an update immediately upon fresh install but `:Lazy update` fails with the message:

```
checkout failed
error: The following untracked working tree files would be overwritten by checkout:
    doc/tags
Please move or remove them before you switch branches.
Aborting
```

I'm not pulling from a release, rather from the top of tree. Perhaps I should be specifying a release. Here is my Lazy entry for monokai-pro:

```lua
  {
    "loctvl842/monokai-pro.nvim",
    lazy = false,
    priority = 1000,
    config = function()
      require("themes.monokai-pro")
    end,
  },
```

It appears `git` thinks my monokai-pro  `doc/tags` file has local modifications but when I diff it with yours it is the same so that is confusing.

**Reply by doctorfree** [reply]
And my `themes.monokai-pro` config (all I am doing unusual here is linking a highlight group for the Alpha dashboard):

```lua
local settings = require("configuration")
local theme = settings.theme
local style = settings.theme_style
local filter = "pro"
if settings.theme == "monokai-pro" then
  if style == "classic" or style == "octagon" or style == "machine" or style == "ristretto" or style == "spectrum" then
    filter = style
  end
end

require("monokai-pro").setup({
  transparent_background = settings.enable_transparent,
  terminal_colors = true,
  devicons = true, -- highlight the icons of `nvim-web-devicons`
  italic_comments = true,
  filter = filter, -- classic | octagon | pro | machine | ristretto | spectrum
  -- Enable this will disable filter option
  day_night = {
    enable = false, -- turn off by default
    day_filter = filter, -- classic | octagon | pro | machine | ristretto | spectrum
    night_filter = "spectrum", -- classic | octagon | pro | machine | ristretto | spectrum
  },
  inc_search = "background", -- underline | background
  background_clear = {
    -- "float_win",
    "toggleterm",
    "telescope",
    "which-key",
    "renamer"
  },-- "float_win", "toggleterm", "telescope", "which-key", "renamer", "neo-tree"
  plugins = {
    -- bufferline = {
    --   underline_selected = false,
    --   underline_visible = false,
    -- },
    -- indent_blankline = {
    --   context_highlight = "default", -- default | pro
    --   context_start_underline = false,
    -- },
    "alpha",
    "cmp",
    "dashboard",
    "gitsign",
    "illuminate",
    "lazy",
    "lsp",
    "mason",
    "neo-tree",
    "noice",
    "notify",
    "nvim-navic",
    "nvim-tree",
    "nvim-treesitter",
    "renamer",
    "scrollbar",
    "telescope",
    "toggleterm",
    "which-key",
    "wilder"
  },
  override = function(c) end,
})

if settings.theme == "monokai-pro" then
  vim.cmd("colorscheme monokai-pro")
  if settings.enable_alpha then
    vim.api.nvim_set_hl(0, "AlphaShortcut", { link = "DashboardShortcut" })
  end
end
```

I'm stumped.

**Comment by doctorfree** [comment]
The issue is resolved in the current plugin master branch. I am closing this discussion. Thanks for a beautiful theme!
